x-pack/plugins/cloud_security_posture/server/routes/compliance_dashboard/get_clusters.ts

```diff
@@ -66,6 +64,9 @@ export const getClustersQuery = (query: QueryDslQueryContainer): SearchRequest =
       },
     },
   },
+  pit: {
```

Comment:
i'm assuming when adding PIT no need to specify the index?

Reply:
Yes, the index is encoded in the PIT ID